File: Lab8/app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button btn_SignIn = findViewById(R.id.btn_main_signIn);
        Button btn_Register = findViewById(R.id.btn_rmain_egister);
        btn_SignIn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, SignActivity.class);
                startActivity(intent);
            }
        });
        btn_Register.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, RegiterActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: Lab3_c/app/src/main/java/com/example/lab3_c/MainActivity.kt
package com.example.lab3_c

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button

class MainActivity : AppCompatActivity()  {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var btnMau = findViewById<Button>(R.id.btnChonMau);
        btnMau.setOnClickListener (View.OnClickListener {
            val intent = Intent(this, ChonMauActivity::class.java)
            startActivity(intent)
        })
    }

}